.github/workflows/cmake.yml

```diff
@@ -84,3 +84,13 @@ jobs:
       shell: bash
       working-directory: ${{github.workspace}}/mdspan-build
       run: make install
+    
+    - name: Test Target Export
+      shell: bash
+      working-directory: ${{github.workspace}}/mdspan-src
+      run: bash ./scripts/test_install.sh . ${{github.workspace}}/mdspan-build
```

Comment:
I am not a fan of that bash script file, I would have just inlined the commands here.

Reply:
I added the bash script instead of inlining because even though it's just a couple of commands we have to run it twice -- one for build-tree and one for install-tree. I figured it would reduce the risk of oversights from changing one of those configurations vs the other, but I'm okay with inlining if you prefer that

Reply:
Could others weigh in on this?